### PR TITLE
Funds Transfer Notifications Added

### PR DESCRIPTION
**Pull request by cattiwill**

### CODE REVIEW

File: PelotonEppSdk/Models/FundsTransferNotificationsRequest.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+using System.Threading.Tasks;
+using PelotonEppSdk.Classes;
+using PelotonEppSdk.Enums;
+
+namespace PelotonEppSdk.Models
+{
+    public class FundsTransferNotificationsRequest : RequestBase
+    {
+        /// <summary>
+        /// The token representing the data and time for which notifications will be provided
```
Comment (dandersonpeloton) [inline PelotonEppSdk/Models/FundsTransferNotificationsRequest.cs:14]: date instead of data

**Comment by dandersonpeloton** [comment]
+1

File: PelotonEppSdk/Models/FundsTransferNotificationsRequest.cs (hunk 1 above)
Comment (fraserd) [inline PelotonEppSdk/Models/FundsTransferNotificationsRequest.cs:14]: ^^

File: PelotonEppSdk/Models/FundsTransferNotificationsTokenRequest.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+namespace PelotonEppSdk.Models
+{
+    public class FundsTransferNotificationsTokenRequest : RequestBase
+    {
+        /// <summary>
+        /// Sets the date associated with the token, from which, the Funds Transfers Notifications will be returned
+        /// </summary>
+        [Required]
+        public string FromDateUtc { get; set; }
```
Comment (fraserd) [inline PelotonEppSdk/Models/FundsTransferNotificationsTokenRequest.cs:16]: The extraneous comma after 'which' ought to be deleted

File: PelotonEppSdk/Models/Response.cs [hunk 3]
```diff
@@ -26,7 +26,7 @@ public class Response
         /// <summary>
         /// This field will return a value for any transaction that occurs, otherwise will be empty.
         /// </summary>
-        public string TransactionRefCode { get; set; }
+        //public string TransactionRefCode { get; set; }
```
Comment (fraserd) [inline PelotonEppSdk/Models/Response.cs:29]: No longer needed -- can delete the comment and the <summary> info.

**Comment by fraserd** [comment]
Nice to get that refactor from Response to TransactionResponse. +1

File: PelotonEppSdkTests/FundsTransferNotificationsTests.cs [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+            var result2 = notificationsRequest.GetAsync().Result;
+            Assert.IsTrue(result2.Success);
+            Assert.AreEqual(0, result2.MessageCode);
+            Assert.IsNotNull(result2.Token);
+
+            // Create a transfer
+            var transfer = GetBasicFundsTransferRequest();
+            transfer.References = null;
+
+            var validationResults = transfer.Validate();
```
Comment (fraserd) [inline PelotonEppSdkTests/FundsTransferNotificationsTests.cs:164]: @cattiwill did you mean to use Validate() or  TryValidate() ?

**Pull request merged**